officemanager/department/service/service.py

```diff
@@ -1 +1,78 @@
-#CRUD operations with DB
+"""
+CRUD operations with DB
+"""
+import requests
+
+def get_departments():
+    """
+    Return list of departments
+    """
+    url = "http://127.0.0.1:8000/api/department/"
+    req = requests.get(url).json()
+    return req
+
+def get_department_by_id(id):
+    """
+    Return info about specified department by id
+    """
+    url = "http://127.0.0.1:8000/api/department/{}/".format(id)
```

Comment:
It's recommended to use f-strings instead of str.format()

Reply:
127.0.0.1:8000 - host and port should be configured via settings.py. Otherwise you need to fix the code before the deployment to different environments